src/controllers/auth.js

```diff
@@ -13,14 +13,8 @@ const { signToken, checkPassword } = encryption;
 class Auth {
   static async login(req, res, next) {
     try {
-      const { email, password } = req.body;
+      const foundUser = req.user;
 
-      const foundUser = await findUser({ email });
-
-      const isMatch = checkPassword(password, foundUser.password);
-      if (!isMatch) {
-        return ErrorResponse.notFoundError(res, 'Invalid credentials');
-      }
```

Comment:
This was already being handled in the verifyLogin middleware in middlewares/login.js